src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -3,3 +3,6 @@ framework:
         storage_id: session.storage.mock_file
     profiler:
         enabled: false
+
+sulu_document_manager:
```

Comment:
I have already create #2601 for that, and as described there I think we can wrap the kernel in a more elegant way without moving a `WebsiteKernel` to our core.

However, that does not solve the issue we have here: If we have to set the configuration here, the `PreviewBundle` might have to know about all the required configuration of all the other bundles. Imagine a `ProductBundle`, `ArticleBundle` and other stuff also need some special configuration for the preview.

Thought about getting the Kernel in the extension, and check there what should be set, would also reduce the configuration effort. Doesn't work, because the kernel doesn't seem to be accessible in the Extension... But I just got another idea, will try to implement that in a different way.